src/predefined/continuous_famous_systems.jl

```diff
@@ -1405,3 +1498,297 @@ end
     du2 = 1/τ*(-N + G)
     return SVector{2}(du1, du2)
 end
+
+"""
+```julia
+sakarya(u0= [-2.8976045, 3.8877978, 3.07465];
+    a =-1.0,
+    b = 1.0,
+    c = 1.0,
+    h = 1.0,
+    p = 1.0,
+    q = 0.4,
+    r = 0.3,
+    s = 1.0
+)
+```
+```math
+\\begin{aligned}
+\\dot{x} &= ax + hy + syz\\\\
+\\dot{y} &= -by - px + qxz \\\\
+\\dot{z} &= cz - rxy
+\\end{aligned}
+```
+A system presenting robust chaos that varies from single wing to double wings to four wings.
+Its attractor arises due to merging of two disjoint bistable attractors [^Li2015].
```

Comment:
Do you remember at which parameter value thius happens\/ and which is the parameter that is actually varied for this to happen?

Also, is it really necessary that wre keep so many free parameters?

Reply:
No I don't, I'll give a read at the paper again and get back at you.

Regarding the free parameters, I'm following Gilpin's [implementation](https://github.com/williamgilpin/dysts/blob/master/dysts/flows.py#L766) and it seems that not only for this system but also for others (the  [`thomas`](https://github.com/williamgilpin/dysts/blob/master/dysts/flows.py#L83) implementation for example) the amount of free parameters exceeds the systems found in the literature. Should I maintain the amount found in the python implementation or follow the literature equations?

Reply:
Follow literature equations!

Reply:
will do!